apps/engine/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@midday/engine",
+  "scripts": {
+    "dev": "wrangler dev src/index.ts",
+    "deploy": "wrangler deploy --minify src/index.ts",
+    "clean": "rm -rf .turbo node_modules",
```

Comment:
`rm -rf` won't support in windows machines. i would be using https://www.npmjs.com/package/rimraf

Just a suggestion